source/_components/pilight.markdown

```diff
@@ -53,6 +53,11 @@ pilight:
       - 42
 ```
 
+## {% linkable_title Known issues %}
+
+- You always have to use the correct types for service data for the `pilight.send` service, as all data is sent unchecked and unchanged to the daemon. If you choose the wrong type, your call gets rejected and all you get is a `insufficient number of arguments` in the pilight log. To retrieve the right argument type, read the [implementation of your protocol](https://github.com/pilight/pilight/tree/master/libs/pilight/protocols).
```

Comment:
It also says that if there was a died pilight daemon or any other problem. The only hint that point that there is something wrong with the actual _message data_ is the error in the pilight log. The log message in the hass log says only unspecific that _something_ went wrong with the service call.

So if the user encounters this error he can just check the pilight error log and knows that he probably hit that known issue. So it is the more specific way to check.